SchoolManagerSystem.Common/DTOs/LoginResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Security.Principal;
+
+namespace SchoolManagerSystem.Common.DTOs
+{
+	public class LoginResponse
+	{
+        public string Id { get; set; }
+        public string FirstName { get; set; }
+		public string LastName { get; set; }
```

Comment:
format the page to align the properties

Reply:
on my laptop, the only thing that is wrong is having unused reference.